src/Model/MethodGo.cs

```diff
@@ -390,11 +399,32 @@ public string HTTPMethodDecorator
             }
         }
 
+        public List<string> SendDecorators

```

Comment:
Looks like you're just following suit, which is reasonable, but mind if we make this an `IEnumerable<string>` instead of a `List<string>`?

Reply:
Fixed